test/residency/prototypes/result_contract.py

```
@@ -0,0 +1,575 @@
+EMPTY_SHA256 = hashlib.sha256(b"").hexdigest()
+
+
+class PrototypeResultError(ValueError):
+    pass
+
+
+def _fail(message: str) -> None:
+    raise PrototypeResultError(message)
+
+
+def _object_from_pairs(pairs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_object_from_pairs`

(ANN202)